api/routes/slot.py

```diff
@@ -28,10 +30,63 @@ def review_slot(user_id):
         update(Slot)
         .where(Slot.user_id == user_id)
         .where(Slot.slot_id == slot_id)
-        .where(Slot.end <= instant())
         .values(feedback=feedback)
     )
     db.session.execute(new_dur)
+
+    # if the session went badly or well, adjust the break period
+    # add on 5 minutes if reacted poorly, take off 5 mins if good
+    # adjust session after as well
+    if feedback == 1 or feedback == 3:
```

Comment:
need to check that this slot has not been reviewed in the past

Reply:
This is left out on purpose (see [6cff2f5](https://github.com/DRP-4/DRP-4/pull/81/commits/6cff2f5786fdfe38fc14d80f533e04191a325c82)) - it's easier to do this check in the frontend

Reply:
Its not, especially in presence of collaboration

Reply:
It is even more easier to just not do the check and just do absolute update instead of a relative one. Then you won't need to do any of the error handling.

The reason frontend check is impossible in presence of collaboration is simple - one client does not know much about other clients using the service, so if one has reviewed the session, other client has no way reliable way of figuring it out.